From: Georg Lehmann <dadschoorse@gmail.com>
Subject: [PATCH] winevulkan: Don't support VK_EXT_headless_surface.
Message-Id: <20201119115652.7225-1-dadschoorse@gmail.com>
Date: Thu, 19 Nov 2020 12:56:52 +0100

It's completly broken because the wine drivers only implement
VK_KHR_win32_surface.

Signed-off-by: Georg Lehmann <dadschoorse@gmail.com>
---
 dlls/winevulkan/make_vulkan | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/winevulkan/make_vulkan b/dlls/winevulkan/make_vulkan
index 11111111111..11111111111 100755
--- a/dlls/winevulkan/make_vulkan
+++ b/dlls/winevulkan/make_vulkan
@@ -92,6 +92,7 @@ UNSUPPORTED_EXTENSIONS = [
     "VK_EXT_headless_surface", # Needs WSI work.
     "VK_KHR_display", # Needs WSI work.
     "VK_KHR_surface_protected_capabilities",
+    "VK_EXT_headless_surface", # Needs WSI work.
     "VK_LUNARG_direct_driver_loading", # Implemented in the Vulkan loader
 
     # Device extensions

-- 
0.0.0

